docs/conversion_examples_gallery/recording/intan.rst

```diff
@@ -0,0 +1,32 @@
+Intan data conversion
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Convert Intan data to NWB using :py:class:`~nwb_conversion_tools.datainterfaces.IntanRecordingInterface`.
+
+.. code-block:: python
+
+    >>> from datetime import datetime
+    >>> from dateutil import tz
+    >>> from pathlib import Path
+    >>> from nwb_conversion_tools import IntanRecordingInterface
+    >>> 
+    >>> # For this data interface we need to pass the location of the `.rhd` file 
+    >>> suffix = "rhd" # This can also be rhs
+    >>> file_path = f"{ECEPHY_DATA_PATH}/intan/intan_{suffix}_test_1.{suffix}"
+    >>> # Change the file_path to the location in your system
+    >>> interface = IntanRecordingInterface(file_path=file_path)
+    >>> 
+    >>> # Extract what metadata we can from the source files
+    >>> metadata = interface.get_metadata()
+    >>> # session_start_time is required for conversion. If it cannot be inferred 
+    >>> # automatically from the source files you must supply one.
+    >>> session_start_time = datetime(2020, 1, 1, 12, 30, 0, tzinfo=tz.gettz("US/Pacific"))
+    >>> metadata["NWBFile"] = dict(session_start_time=session_start_time)
```

Comment:
```suggestion
    >>> metadata["NWBFile"].update(session_start_time=session_start_time)
```

can you please try to do it this way?

Reply:
Does intan already contain the datetime in the header?

Reply:
Intan headers have a wealth of information, particularly about electrode metadata. But I don't think we extract much of it automatically just yet

Reply:
Hey, @bendichter, are you reading my comments above that this does not work when the metadata does not contain the key "NWBFile", already? 

Maybe I am missing something?

Reply:
yeah I missed that earlier comment. OK, I see, this is correct. After we get this through let's look at ways we can pull this metadata from the source files.